docs/topics/db/queries.txt

```diff
@@ -581,14 +580,34 @@ that were published in 2008. The entries selected by the second filter may or
 may not be the same as the entries in the first filter. We are filtering the
 ``Blog`` items with each filter statement, not the ``Entry`` items.
 
-All of this behavior also applies to
-:meth:`~django.db.models.query.QuerySet.exclude`: all the conditions in a
-single :meth:`~django.db.models.query.QuerySet.exclude` statement apply to a
-single instance (if those conditions are talking about the same multi-valued
-relation). Conditions in subsequent
-:meth:`~django.db.models.query.QuerySet.filter` or
-:meth:`~django.db.models.query.QuerySet.exclude` calls that refer to the same
-relation may end up filtering on different linked objects.
+.. note::
+
+    At present, this behaviour is not supported for
+    :meth:`~django.db.models.query.QuerySet.exclude`.  Instead, the
+    the arguments in a single
+    :meth:`~django.db.models.query.QuerySet.exclude` call will behave as
+    if they had been chained together in two
+    :meth:`~django.db.models.query.QuerySet.exclude` calls.
+
+    This means that the following two expressions will both return
+    the same results, i.e. all the blogs that do not contain either
+    entries with *"Lennon"* in the headline or entries published in 2008::
+
+        Blog.objects.exclude(entry__headline__contains='Lennon',
+            entry__pub_date__year=2008)
+
+        Blog.objects.exclude(entry__headline__contains='Lennon').exclude(
+            entry__pub_date__year=2008)
+
+    To select all blogs that do not contain entries published with
+    *"Lennon"* that were published in 2008, you would need to make
+    two queries::
+
+        Blog.objects.exclude(
+            entry=Entry.objects.filter(
+                headline__contains='Lennon', pub_date__year=2008).values('id')
```

Comment:
Can this query not be generated using `Q` objects?
`Blog.objects.filter(~Q(entry__headline__contains='Lennon') & ~Q(entry__pub_date__year=2008))`